src/Cache/CacheEngineInterface.php

```diff
@@ -66,4 +66,19 @@ public function decrement(string $key, int $offset = 1);
      * @return bool
      */
     public function clearGroup(string $group): bool;
+
+    /**
+     * Delete a key from the cache asynchronously
+     *
+     * @param string $key Identifier for the data
+     * @return bool True if the value was successfully deleted, false if it didn't exist or couldn't be removed
+     */
+    public function deleteAsync(string $key): bool;
```

Comment:
We can't change the interface in a minor release. Instead add `@method` annotations for these new methods for the time being.